src/app/canvass/[areaAssId]/area/[areaId]/page.tsx

```diff
@@ -22,8 +23,8 @@ export default async function Page({ params }: PageProps) {
   try {
     await apiClient.get<ZetkinOrganization>(`/api/users/me`);
 
-    return <CanvassPage areaAssId={areaAssId} />;
+    return <CanvassPage areaAssId={areaAssId} areaId={areaId} />;
   } catch (err) {
-    return redirect(`/login?redirect=/canvass/${areaAssId}/map`);
+    return redirect(`/login?redirect=/canvass/${areaAssId}/area`);
```

Comment:
I think that the URL should be `areas` (plural). Also, shouldn't this URL contain the `areaId`, so that the user comes back to the exact URL after logging in?